standalone.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-env node */
+"use strict";
+require("./index.js");
+var electron = require("electron");
+var app = electron.app;
+var BrowserWindow = require("electron").BrowserWindow;
+
+app.on("ready", function () {
+    var win = new BrowserWindow({width: 1200, height: 800});
+    win.loadURL("http://localhost:8080/editprefs/alice");
+    win.webContents.openDevTools();
+});
+
+app.on("window-all-closed", function () {
+    app.quit();
```

Comment:
Thus quyte we folk that doon us displesances
http://www.librarius.com/canttran/pardtale/pardtale103-136.htm

Reply:
intense...